plugin/execute_command.py

```diff
@@ -14,6 +15,18 @@ def run(self,
             command_args: Optional[List[Any]] = None,
             session_name: Optional[str] = None,
             event: Optional[dict] = None) -> None:
+        # Handle VSCode-specific command for triggering AC/sighelp
+        if command_name == "editor.action.triggerSuggest":
+            # Triggered from set_timeout as suggestions popup doesn't trigger otherwise.
+            return sublime.set_timeout(lambda: self.view.run_command("auto_complete"))
+        if command_name == "editor.action.triggerParameterHints":
+
```

Comment:
I thought that we had a `lsp_show_signature` command so we could have just 
`return sublime.set_timeout(lambda: self.view.run_command("lsp_show_signature", {"manual": False}))`
but I guess we don't have such a command :)